draft-ietf-quic-transport.md

```diff
@@ -1093,6 +1093,14 @@ protection for the reasons described in Section 9.3 of {{QUIC-TLS}}. An
 efficient algorithm for duplicate suppression can be found in Section 3.4.3 of
 {{?RFC2406}}.
 
+Packet numbers SHOULD be incremented by one. Parallel processing and optimistic
+ACK attack mitigation {{optimistic-ack-attack}} can lead to gaps. The difference
+between the largest sent and the largest acknowledged packet number
+SHOULD be no more than twice the actual number of packets sent.  The number of
+gaps and their sizes SHOULD be kept to a minimum.  A peer MAY close a connection
+with the error BAD_LINK when the observed gap count or loss rate based on packet
```

Comment:
If we add an error code, I'd like to be specific about what it means.  ie: too many gaps?   BAD_LINK doesn't tell me if the connection is too poor to continue with or if there are too many gaps, which are potentially quite different.

Reply:
We could add a note about "SHOULD add a reason phrase" when using the BAD_LINK error.

Reply:
I'm not a fan of adding a separate error here. There are a lot of places where an implementation might introduce some limit as a DoS defense (e.g. too many gaps in STREAM data), many of those implementation specific. BAD_LINK certainly isn't appropriate in many of those cases.

Reply:
Which error would you use then? Or would you silently drop? I considered that as well.

Reply:
I agree with @marten-seemann here. I think if an implementation has a specific limit/threshold, it should just silently drop.

Reply:
The reason I chose not to silently drop was the subsequent reset that would follow.